scripts/kubernetes/frontend_deployment.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: funcatron-frontend
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: funcatron-frontend
+    spec:
+      containers:
+        - name: frontend
+          resources:
+            requests:
+              cpu: 0.4
+              memory: 256Mi
+          #nodeSelector:
```

Comment:
I added this nodeSelector to match `"acceptedResourceRoles": ["slave_public"]` in Mesos, but left it commented out since most people won't have a node with the right label in their cluster.